tests/test_san_joaquin_valley_air_pollution.py

```
@@ -76,21 +80,15 @@ def test_location():
 def test_source():
     assert (
         parsed_items[0]["source"]
-        == "https://www.valleyair.org/Board_meetings/GB/GB_meetings_2022.htm"
+        == "https://www.valleyair.org/public-meetings-and-participation/governing-board/?year=2022"  # noqa
```

Ruff (0.14.8)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)